src/Event/EventBundle/Controller/Backend/MediaController.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+namespace Event\EventBundle\Controller\Backend;
+
+use Symfony\Component\Config\Definition\Exception\Exception;
+use Symfony\Component\HttpFoundation\Request;
+use Event\EventBundle\Controller\Controller;
+use Event\EventBundle\Entity\Media;
+use Event\EventBundle\Form\Type\MediaType;
+
+class MediaController extends Controller
+{
+
+    public function getUploadPath()
+    {
+        return $this->get('kernel')->getRootDir() . DIRECTORY_SEPARATOR . '..' . DIRECTORY_SEPARATOR .
+            'web' . DIRECTORY_SEPARATOR . 'uploads' . DIRECTORY_SEPARATOR . 'media' . DIRECTORY_SEPARATOR;
+    }
+    
+    public function getUploadUrl(){
+        return '/uploads/media/';
+    }
+
+    public function indexAction()
+    {
+        return $this->render('EventEventBundle:Backend/Media:index.html.twig', array(
+            'media' => $this->getRepository('EventEventBundle:Media')->findAll(),
+            'uploadPath' => $this->getUploadPath(),
+            'uploadUrl' => $this->getUploadUrl()
+        ));
+    }
+
+    public function manageAction(Request $request, $id = null)
+    {
+        $oldFileName = '';
+        if ($id === null) {
+            $entity = new Media();
```

Comment:
`$entity` is not yet defined, as well use user-friendly naming, not entity but `$media`